packages/runtime/src/utils/dom.ts

```diff
@@ -0,0 +1,92 @@
+import { isStaticResourcesEqual, safeWrapper } from './tool';
+
+export function createScript(
```

Comment:
@RussellCanfield  @2heal1 in future I think i can move some of federation/node into a /node.ts file here, which provides the same api but not to DOM script loader.

Reply:
yes~